lib/rubycritic.rb

```diff
@@ -1,22 +1,7 @@
-require "rubycritic/configuration"
+require "rubycritic/command_factory"
 
 module Rubycritic
   def self.create(options = {})
```

Comment:
:+1: on that. This part of Rubycritic always felt more an CLI abstraction layer rather than a programatically API, so I think we should refactor it as we want it - The `AnalysersRunner` sounds more like the way to go if someone wants an API (which is what we do in an internal app :smile:)